setup.py

```diff
@@ -6,7 +6,7 @@
     long_description = f.read()
 
 setup(name="pipelinewise-target-snowflake",
-      version="1.6.6",
```

Comment:
can you please leave the original version? We'll bump the version in a separated PR

Reply:
@koszti 👍  done. Sorry about that, I guess its a habit 😄